tests/components/matter/fixtures/config_entry_diagnostics_redacted.json

```diff
@@ -0,0 +1,4243 @@
+            "cluster_id": 1030,
+            "cluster_type": "chip.clusters.Objects.OccupancySensing",
+            "cluster_name": "OccupancySensing",
+            "attribute_id": 65532,
+            "attribute_type": "chip.clusters.Objects.OccupancySensing.Attributes.FeatureMap",
+            "attribute_name": "FeatureMap",
+            "value": 0
+          },
+          "1/1030/65533": {
+            "node_id": 5,
+            "endpoint": 1,
+            "cluster_id": 1030,
+            "cluster_type": "chip.clusters.Objects.OccupancySensing",
+            "cluster_name": "OccupancySensing",
+            "attribute_id": 65533,
+            "attribute_type": "chip.clusters.Objects.OccupancySensing.Attributes.ClusterRevision",
+            "attribute_name": "ClusterRevision",
+            "value": 3
+          },
+          "1/1030/65528": {
+            "node_id": 5,
+            "endpoint": 1,
+            "cluster_id": 1030,
+            "cluster_type": "chip.clusters.Objects.OccupancySensing",
+            "cluster_name": "OccupancySensing",
+            "attribute_id": 65528,
+            "attribute_type": "chip.clusters.Objects.OccupancySensing.Attributes.GeneratedCommandList",
+            "attribute_name": "GeneratedCommandList",
+            "value": []
+          },
+          "1/1030/65529": {
+            "node_id": 5,
+            "endpoint": 1,
+            "cluster_id": 1030,
+            "cluster_type": "chip.clusters.Objects.OccupancySensing",
+            "cluster_name": "OccupancySensing",
+            "attribute_id": 65529,
+            "attribute_type": "chip.clusters.Objects.OccupancySensing.Attributes.AcceptedCommandList",
+            "attribute_name": "AcceptedCommandList",
+            "value": []
+          },
+          "1/1030/65531": {
+            "node_id": 5,
+            "endpoint": 1,
+            "cluster_id": 1030,
+            "cluster_type": "chip.clusters.Objects.OccupancySensing",
+            "cluster_name": "OccupancySensing",
+            "attribute_id": 65531,
+            "attribute_type": "chip.clusters.Objects.OccupancySensing.Attributes.AttributeList",
+            "attribute_name": "AttributeList",
+            "value": [0, 1, 2, 65528, 65529, 65531, 65532, 65533]
+          }
+        },
+        "endpoints": [0, 1],
+        "root_device_type_instance": {
+          "__type": "<class 'matter_server.common.models.device_type_instance.MatterDeviceTypeInstance'>",
+          "repr": "<MatterDeviceTypeInstance RootNode (N:5, E:0)>"
+        },
+        "aggregator_device_type_instance": null,
+        "device_type_instances": [
+          {
+            "__type": "<class 'matter_server.common.models.device_type_instance.MatterDeviceTypeInstance'>",
+            "repr": "<MatterDeviceTypeInstance DimmableLight (N:5, E:1)>"
+          }
+        ],
+        "node_devices": [
+          {
+            "__type": "<class 'matter_server.common.models.node_device.MatterNodeDevice'>",
```

Comment:
Can we expand these classes to either just the `repr` value or we add `dataclasses.as_dict` on it ? ?